pkg/oc/clusteradd/componentinstall/apply_template.go

```diff
@@ -130,7 +130,7 @@ func (opt installReadyTemplate) Install(dockerClient dockerhelper.Interface) err
 		return nil
 	}
 
-	if err := wait.PollImmediate(time.Second, 5*time.Minute, opt.template.WaitCondition); err != nil {
+	if err := wait.PollImmediate(time.Second, 20*time.Minute, opt.template.WaitCondition); err != nil {
```

Comment:
I don't like this, but the openshift-logging roles in openshift-ansible take about 8 minutes to run on a pretty fast workstation so we blow past the original 5 minute setting. Is it possible to set a timeout just for openshift-logging?

Alternatively I guess we could just ensure the job exists and move on, which would happen rather quickly.

Look forward to input on a better alternative.